### ISSUE TITLE
Unity Arcore Severe Objects drift (Planes and Objects)

### ISSUE THREAD

**Issue by camilorodriguezapriorins** [google-ar/arcore-unity-sdk#248]
Good morning

We have been working with ARcore since this year in February. Currently, we are working with 1.1.0 version (we are planing to update to Arcore 1.2 this week). With this version, in our app, we are experiencing severe drift when loading the scene (at the very beginning). This happens sometimes. In order to obtain a mesure of the drift, we draw a cube and 3 lines representing the 3 axes and the 0,0,0 point of the environment. We realized before starting to scan surfaces, these objects move a lot. After scanning surfaces and creating new 3d models attached to the anchors in the planes, the stability improves a lot (the drift is not happening so much when the surface is big enough). Nevertheless, we depend on the 0,0,0 point we draw at the beginning for drawing the entire scene after scanning the surfaces. Someone could please help us with some idea? Basically, we want the 0,0,0 point don't moves so much at the beginning in order to keep the first point the camera see when the ARcore unity scene is loaded as the 0,0,0 point. Note: We sometimes also experience drift in the planes after the surfaces have been scanned, but this is less frequent than the drift at the beginning.

**Comment by pablisho** [comment]
Hi, I think the initial behaviour is expected. First, it takes some initial time to start the session (it is not traking until the session status says so, so it's not drift), and then it may improve general tracking after some features are recognized more than once.
If you need a reference frame that is related to the real world, I think you could try [AugmentedImages](https://developers.google.com/ar/develop/unity/augmented-images/). Let me know if it helps.

**Comment by camilorodriguezapriorins** [comment]
Thanks for your answer @pablisho , it helped us a lot. Now we are using the Augmented Image feature and it looks like could work for what we want.

**Comment by pablisho** [comment]
@camilorodriguezapriorins I'm closing this as I believe your question was answered. Feel free to reopen if you still experience issues with this or open another issue if you have more questions.
